iroha/src/account.rs

```diff
@@ -132,6 +136,10 @@ pub mod isi {
         ),
         /// Variant of the generic `Add` instruction for `PublicKey` --> `Account`.
         AddSignatory(<Account as Identifiable>::Id, PublicKey),
+        /// Variant of the generic `Add` instruction for `Instruction` --> `Account`.
+        AddInstruction(<Account as Identifiable>::Id, Box<Instruction>),
+        /// Variant of Iroha Special Instructions executes an ISI associated with the account.
+        ExecuteInstruction(<Account as Identifiable>::Id),
```

Comment:
Then both - AddInstruction and ExecuteInstruction will be a composition or alias of out-of-the-box MintAsset instruction.

Reply:
@humb1t you mean not to add these as separate ISIs?